exporter/exporterhelper/queue_batch.go

```diff
@@ -0,0 +1,47 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package exporterhelper // import "go.opentelemetry.io/collector/exporter/exporterhelper"
+
+import (
+	"go.opentelemetry.io/collector/exporter/exporterhelper/internal"
+	"go.opentelemetry.io/collector/exporter/exporterhelper/internal/queuebatch"
+	"go.opentelemetry.io/collector/exporter/exporterhelper/internal/request"
+	"go.opentelemetry.io/collector/exporter/exporterqueue"
+)
+
+// QueueConfig defines configuration for queueing batches before sending to the consumerSender.
+type QueueConfig = exporterqueue.Config
+
+// Deprecated: [v0.123.0] use WithQueueBatch.
+func WithRequestQueue(cfg exporterqueue.Config, encoding exporterqueue.Encoding[Request]) Option {
+	return WithQueueBatch(cfg, QueueBatchSettings[Request]{Encoding: encoding})
+}
+
+// QueueBatchConfig defines configuration for queueing and batching for the exporter.
+type QueueBatchConfig = exporterqueue.Config
```

Comment:
For the moment still use the old queue Config until we do the transition to the newly defined Config. We will do a slow move to that.